src/WebJobs.Script.Scaling/AppServiceSettings.cs

```diff
@@ -249,6 +250,28 @@ public static string LockHandleId
             }
         }
 
+        /// <summary>
+        /// Gets or sets a runtime encryption key
+        /// </summary>
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
```

Comment:
nit: Suppressions are usually defined in the `GlobalSupressions` file.

Reply:
Fixed.